tests/test_tracer.py

```diff
@@ -1057,3 +1059,49 @@ def test_enable(monkeypatch):
     monkeypatch.setenv("DD_TRACE_ENABLED", "false")
     t2 = ddtrace.Tracer()
     assert not t2.enabled
+
+
+def test_runtime_id_parent_only():
```

Comment:
did you mean to include those here since there's no change to the tracer?

Reply:
Ah, I missed reverting this. I mean it's not bad that extra test coverage is included I guess 😛